tests/sqllogictests/suites/udf_server/user_functions_table.test

```diff
@@ -42,21 +42,11 @@ statement ok
 CREATE FUNCTION add_float (FLOAT, DOUBLE) RETURNS DOUBLE LANGUAGE python HANDLER = 'add_float' ADDRESS = 'http://0.0.0.0:8815';
 
 # TEST: SHOW USER FUNCTIONS
-query 
+statement ok
 SHOW USER FUNCTIONS LIKE 'add%';
-----
-add_float NULL (empty) {"arg_types":["Float32 NULL","Float64 NULL"],"return_type":"Float64 NULL"} python
-add_signed NULL (empty) {"arg_types":["Int8 NULL","Int16 NULL","Int32 NULL","Int64 NULL"],"return_type":"Int64 NULL"} python
-add_unsigned NULL (empty) {"arg_types":["UInt8 NULL","UInt16 NULL","UInt32 NULL","UInt64 NULL"],"return_type":"UInt64 NULL"} python
-
 
-query 
+statement ok
 SHOW USER FUNCTIONS WHERE LANGUAGE = 'SQL';
-----
```

Comment:
Because add time column will cause the test failed. 

Please ref this test
https://github.com/datafuselabs/databend/blob/f5598f87180cfa3d13ee849ba3d6e4e52f4bf0b5/tests/suites/0_stateless/12_time_travel/12_0002_time_travel_show_tables_hist.sql#L7 and add it in there.

Reply:
https://github.com/datafuselabs/databend/pull/14540/commits/6cbd2ea89ded8157613f82413e5c642acdec495c

done